### PR TITLE
#162407187 Create seeders for different environments

### PR DESCRIPTION
**Pull request by janetnim**
#### What does this PR do?
Create seeders for the following environments:
- end to end tests
- unit tests
- Also a `default` seeder for attributes used across all environments.

#### Description of Task to be completed?
1. Create separate folders for different seeders environments
2. Edit db:seed to be environment specific
3. Configure sequlize to create environment specific seeders files

#### How should this be manually tested?
1. Clone the repository `https://github.com/andela/travel_tool_back`
2. Change to the travel_tool directory on your editor and run the following commands:
* git checkout `ch-seeders-for-different-environments-162407187` in travel_tool_back 
* `yarn install` 
* `make start` 
3. To run seeded data for different environments type in:
* **yarn db:seed** or **make seed** -> for default
* **yarn db:seed:unittest** or **make seed-unittest** -> for unittests
* **yarn db:seed:ete** or **make seed-e2e**-> for end to end tests
4. To create a seeder file for a specific environment, specify the `--seeders-path` as seen in the snippet below:
**`sequelize seed:generate --name demo_user --seeders-path src/database/seeders/e2e`**
If the seeder file is to be used across all environments, do not specify the --seeders-path and the file will be created in the default seeders folder.
**`sequelize seed:generate --name demo_user`**

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[162407187](https://www.pivotaltracker.com/story/show/162407187)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -108,7 +108,7 @@ ifeq ($(CONTAINER),)
 	$(call container_err)
 else
 	${INFO} "Running travela seeders"
-	@docker-compose -f $(DOCKER_DEV_COMPOSE_FILE) exec app yarn db:seed
+	@docker-compose -f $(DOCKER_DEV_COMPOSE_FILE) exec app yarn db:seed-dev
```
Comment (HarithJ) [inline Makefile]: Wouldn't I also need the default seeders?

Comment (janetnim) [reply Makefile]: I thought that when we run `make seed` we only want to seed for development.

Comment (HarithJ) [reply Makefile]: Shouldn't the default seeders be included across all environments?

Comment (ronaldndirangu) [reply Makefile]: Cool making that change

Comment (janetnim) [reply Makefile]: @HarithJ So when I run make seed should I be able to seed all seeder files or those of a specific environment?

Comment (janetnim) [reply Makefile]: @HarithJ What if I create different make seed commands for each environment as seen below:
screen shot 2018-12-07 at 16 59 38
So the default will still be `make seed`.

Comment (HarithJ) [reply Makefile]: @janetnim, This make file is only for development purposes, thus there is no need to have `seed-prod` and the others, only the **default** and the **development** seeders should be executed.

Comment (janetnim) [reply Makefile]: @HarithJ So can we have `make seed` and make seed-dev` instead?

Comment (HarithJ) [reply Makefile]: Or, only have `make seed` which executes both; the default seeders and the development seeders.

**Pull request merged**